packages/venia-ui/lib/RootComponents/Category/category.css

```diff
@@ -50,14 +50,18 @@
     padding-bottom: 1.5rem;
 }
 
+.contentWrapper {
+    width: 100%;
+}
+
 .sidebar {
     display: none;
 }
 
 @media (min-width: 1024px) {
-    .root {
+    .contentWrapper {
         display: flex;
-        flex-wrap: wrap;
+        flex-wrap: nowrap;
```

Comment:
Tiny nitpick, but this should be the default so this line is redundant; consider removing.

```suggestion
```

Reply:
@tjwiebell Good find! I've removed the value and it's still working.